src/docs/build-deploy-and-maintain-apps/maintain-an-application.md

```diff
@@ -358,7 +358,7 @@ Once your application is running in OpenShift, you can use [Sysdig](https://digi
 
 1.  [Onboard to Sysdig](/sysdig-monitor-onboarding/) and setup the OpenShift project set access to monitor your applications
 2. [Create Sysdig monitoring dashboards](https://docs.sysdig.com/en/docs/sysdig-monitor/dashboards/) to gather important application metrics such as resource utilization and service Golden Signals
-3. Merely having a monitoring dashboard isn't sufficient since you won't be actively checking it throughout the day. Instead, it's advisable to establish alerts for monitoring metrics. This approach ensures that you receive notifications when potential issues arise, allowing you to address problems in their early stages. Follow the guide on [how to setup Sysdig alerts via Rocket.Chat](/app-monitoring/sysdig-monitor-create-alert-channels/)
+3. Merely having a monitoring dashboard isn't sufficient since you won't be actively checking it throughout the day. Instead, it's advisable to establish alerts for monitoring metrics. This approach ensures that you receive notifications when potential issues arise, allowing you to address problems in their early stages. Follow the guide on [how to setup Sysdig alerts via Rocket.Chat](/sysdig-monitor-create-alert-channels/)
```

Comment:
Don't think this link format will work? Also, the other link further down this page is broken using this same format.